src/frontends/pytorch/src/op/min_max.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+std::shared_ptr<Node> get_all_axes(NodeContext& context) {
```

Comment:
please add this function to nameless namespace

Reply:
moved to utils and reused in aten::sum